test/modules/core/views/view.spec.ts

```diff
@@ -19,6 +19,26 @@ test('View#imports', t => {
   t.end();
 });
 
+test('View#clone', t => {
+  const view = new MapView({
+    id: 'test-view',
+    latitude: 0,
+    longitude: 0,
+    zoom: 1
+  });
+  const clone = view.clone({
+    id: 'cloned-view',
+    zoom: 5
+  });
+  t.ok(clone instanceof MapView, 'clone returns an instance of MapView');
+  t.ok(clone !== view, 'clone returns a new instance');
+  t.is(clone.id, 'cloned-view', 'clone id is overridden');
+  t.is(clone.props.zoom, 5, 'clone prop zoom is overridden');
+  t.is(clone.props.latitude, view.props.latitude, 'other props are preserved');
+  t.is(clone.props.longitude, view.props.longitude, 'other props are preserved');
```

Comment:
Can we also test that cloning a view should produce an identical view with matching matrices?

Reply:
> Can we also test that cloning a view should produce an identical view with matching matrices?

I considered it, but seemed to require a bit more work than I would like, I feel that adding the equal test should be pretty solid.